react/tic-tac-toe/src/app/index.js

```diff
@@ -5,14 +5,15 @@ import { useDispatch, useSelector } from 'react-redux';
 import authActions from '~redux/auth/actions';
 
 import Routes from './components/Routes';
+
 import '../scss/application.scss';
 
 function App() {
   const { isAuthed } = useSelector(state => state.auth);
   const dispatch = useDispatch();
 
   useEffect(() => {
-    dispatch(authActions.setUp());
+    dispatch(authActions.getLocalStorageValue());
```

Comment:
Why? I liked the `.setUp()` name better... 🥺

Reply:
I was told that setUp name was not descriptive.